dotnet/src/SemanticKernel.Core/Functions/SemanticFunction.cs

```diff
@@ -305,9 +305,7 @@ public ISKFunction SetAIService(Func<ITextCompletion> serviceFactory)
         }
         else
         {
-            var serviceSelector = new DelegatingAIServiceSelector();
-            serviceSelector.ServiceFactory = serviceFactory;
-            this._serviceSelector = serviceSelector;
+            this._serviceSelector = new DelegatingAIServiceSelector { ServiceFactory = serviceFactory };
```

Comment:
nit formatting (apply for others)

```suggestion
            this._serviceSelector = new DelegatingAIServiceSelector 
            { 
                ServiceFactory = serviceFactory 
            };
```